bosh-stemcell/spec/support/stemcell_opensuse_shared_examples.rb

```diff
@@ -0,0 +1,134 @@
+shared_examples_for 'a openSUSE stemcell' do
+
+  describe command('ls -1 /lib/modules | wc -l') do
+    its(:stdout) {should eq "1\n"}
+  end
+
+  context 'installed by image_install_grub', exclude_on_warden: true do
```

Comment:
This exclusion breaks the tests in `cis_spec.rb` and `stig_spec.rb`. Removing it will cause those tests to pass.